Features

punixcorn opened the pull request:
testing 
more comments 
changed formmatter

punixcorn committed:
added a bash script to install packer nvim

punixcorn committed:
changed formatter

added comments to init.lua

Moonixp commented on .installPacker.sh:
make this an installer script ?

punixcorn replied:
sure

Moonixp commented on lua/plugs.lua:
Telescope and formatting still not working

Moonixp reviewed (changes requested):
make installer script & fix telescope and formatting and we can merge

punixcorn committed:
Fixed all issues

updated the script to install the config

using Neoformat instead of formatter

replaced git gutter with vgit

added markdown reader

added color viewer

added nvim tree

added Discord ipc

clangd formatter

more colorschemes

improved comments and changed some keymaps and settings

Moonixp commented on lua/keymap.lua:
```diff
@@ -27,18 +30,19 @@ vim.keymap.set("v", "p", '"_dP')
-vim.keymap.set("n", "<Right>", ":echo 'Use l'<CR>")
-vim.keymap.set("n", "<Up>", ":echo 'Use k'<CR>")
-vim.keymap.set("n", "<Down>", ":echo 'Use j'<CR>")
+vim.keymap.set("n", "<Left>", ":echo 'Use h ya twat'<CR>")
```
really?

punixcorn replied:
yes

Moonixp commented on plugin/nvimtree.lua:
```diff
@@ -0,0 +1,6 @@
+-- Nvim Tree
+vim.g.loaded_netrw = 1 -- required
+vim.g.loaded_netrwPlugin = 1 -- required
+vim.opt.termguicolors = true -- required
```
some of these are already in settings.lua and could have put the rest there

punixcorn replied:
ok...

Moonixp commented on plugin/null-ls.lua:
didn't say anything about null-ls

punixcorn replied:
forgot, not needed, but does not interfere

Moonixp reviewed (approved):
ok adding it
